lib/core/src/server/manager/manager-config.js

```diff
@@ -36,6 +36,9 @@ const getAutoRefs = async (options) => {
   return list.filter(Boolean);
 };
 
+// strip trailing slash from composition URL
```

Comment:
this is obvious from reading the code. WHY does it need to do that?

Reply:
Because the code joining the basePath & `iframe.html` and `stories.json` is naive and assumes the url is lacking the trailing `/`.